apps/estuary/test/integration/event_table_test.exs

```diff
@@ -20,4 +20,24 @@ defmodule Estuary.EventTableTest do
     assert expected_value == actual_value
     assert expected_table_value == actual_table_value
   end
+
+  test "insert_event inserts an event into the #{@event_stream_table_name} table" do
+    aloha_events =
+      Prestige.execute(
+        "SELECT COUNT(*) from #{@event_stream_table_name} WHERE author = 'Steve Aloha' and create_ts = 5 and data = 'some data' and type = 'some type'"
+      )
+      |> Prestige.prefetch()
+
+    assert aloha_events == [[0]]
+    Estuary.EventTable.insert_event("Steve Aloha", 5, "some data", "some type")
+
+    aloha_events =
```

Comment:
Can we avoid reassigning to the same variable in the function, we may put some name which serves the purpose

Reply:
Good call Sukaant only a _monster_ would name shadow their variables.



@jmorris-pillar @jmorris-pillar @jmorris-pillar @jmorris-pillar @jmorris-pillar @jmorris-pillar